app/controllers/base_controller.rb

```diff
@@ -1,3 +1,7 @@
+require 'spree/core/controller_helpers'
```

Comment:
I dont think we need this line and line 9 below.
the 4 helpers in ControllerHelpers are included individually.